types/prismjs/prismjs-tests.ts

```diff
@@ -1,4 +1,4 @@
-import * as Prism from 'prismjs';
```

Comment:
Is this pattern still supported or is this a breaking change?

Reply:
Hi. The pattern isn't supported but I need to find a different way how to do that because this pattern isn't supported too `const Prism = require('prismjs');`

Reply:
Now are supported both patterns
`import * as Prism from 'prismjs';`
`import Prism = require('prismjs');`
but the new variables are writable only if I use `import Prism = require('prismjs');`